public/docs/getting_started/project.md

```diff
@@ -1,31 +1,27 @@
-# Create a Project Folder
+# Creating a project
 
-First create a project folder. You can do this through your favorite file explorer or by running the following on the command line:
+To use most Truffle commands, you need to run them against an existing Truffle project. This section will show you how to create a new project.
 
-```none
-$ mkdir myproject
-```
-
-# Initialize Your Project
+(For those just getting started, we have many boilerplates and example applications available to you through [Truffle Boxes](/boxes). You can use the `truffle unbox <box-name>` command to download these applications.)
 
-Next, initialize your Truffle project by performing the following in the command line:
+First, create a project directory and move into it. You can do this through your favorite file explorer or by running the following on the command line:
```

Comment:
Since you mentioned Truffle boxes, perhaps say something like `To start from scratch, ...` to signal this is a different way of getting started?